v5/core/base_service_test.go

```diff
@@ -747,7 +815,7 @@ func TestRequestBasicAuth2(t *testing.T) {
 	assert.NotNil(t, service)
 	assert.NotNil(t, service.Options.Authenticator)
 
-	var foo *Foo
+	var foo []byte
 	_, err = service.Request(req, &foo)
```

Comment:
```suggestion
	_, err = service.Request(req, nil)
```
For this test, it looks like we're not actually sending a response body back, so should probably just pass in nil for the result param to the Request() method, no?